src/hooks/useFilteredTodos.ts

```diff
@@ -0,0 +1,25 @@
+import { useMemo } from 'react';
+import { FilterType } from '../typedefs';
+import { getFilteredTodos } from '../helpers';
+import { Todo } from '../types/Todo';
+
+interface Options {
+  todos: Todo[]
+}
+
+export const useFilteredTodos = (options: Options) => {
+  const { todos } = options;
+
+  const activeTodos = useMemo(() => (
+    getFilteredTodos(todos, FilterType.ACTIVE)
+  ), [todos]);
+
+  const completedTodos = useMemo(() => (
+    getFilteredTodos(todos, FilterType.COMPLETED)
+  ), [todos]);
```

Comment:
Better to use `reduce` method to decrease complexity when u wanna split the array into 2 arrays))